modules/shared/k8shelpers/connection-manager.go

```diff
@@ -230,6 +231,26 @@ func (cm *DesktopConnectionManager) GetDefaultNamespace(kubeContext string) stri
 	return context.Namespace
 }
 
+// GetNamespaceList returns a list of all namespaces in the cluster
```

Comment:
The `ConnectionManager` handles only connection-related things so it isn't the best place for a namespace convenient method.